src/lib/duration/humanize.js

```diff
@@ -60,6 +63,11 @@ export function getSetRelativeTimeThreshold (threshold, limit) {
     if (limit === undefined) {
         return thresholds[threshold];
     }
+    if (threshold === 's' && !secondsThresholdChanged) {
```

Comment:
This is a bit tricky. I'd drop the variable, always set `ss` to `limit - 1`, so if you need to use it you need to set `ss` last (alone or after `s`, which makes sense).